jib-cli/src/integration-test/java/com/google/cloud/tools/jib/cli/JarCommandTest.java

```diff
@@ -87,16 +89,29 @@ public void testStandardJar_explodedMode_toDocker()
     Path jarPath = Paths.get(Resources.getResource("jarTest/standard/jarWithCp.jar").toURI());
     Integer exitCode =
         new CommandLine(new JibCli())
-            .execute("jar", "--target", "docker://exploded-jar", jarPath.toString());
-    String output = new Command("docker", "run", "--rm", "exploded-jar").run();
-    try (JarFile jarFile = new JarFile(jarPath.toFile())) {
-      String classPath =
-          jarFile.getManifest().getMainAttributes().getValue(Attributes.Name.CLASS_PATH);
-
-      assertThat(classPath).isEqualTo("dependency1.jar directory/dependency2.jar");
+            .execute(
+                "jar",
+                "--target",
+                "docker://" + dockerHost + ":5000/exploded-jar",
```

Comment:
Hmm... I have a doubt. Since it's directly pushing to a Docker daemon, I think the name shouldn't matter. Also, `docker run` isn't supposed to make a network connection, since the image should already exist in the daemon cache. But I'm not sure what's going on.

Reply:
Hmm... on second thought, it doesn't make sense, because the image including the base image layers is already built and pushed to a Docker daemon by Jib.